frame/alliance/README.md

```diff
@@ -65,4 +65,4 @@ to update the Alliance's rule and make announcements.
 
 #### Root Calls
 
-- `init_founders` - Initialize the founding members.
+- `force_set_members` - Initialize the founding members.
```

Comment:
@muharem 
bellow, there is a different desc: `force_set_members` - Set the members via chain governance.`,
please, align both :)

Reply:
done